README.md

```diff
@@ -25,31 +27,224 @@ export default function Example() {
   return (
     <Card>
       <Input placeholder="Type here..." />
-      <Button>Click me</Button>
+      <Button variant="default">Click me</Button>
     </Card>
   );
 }
 ```
 
+## 🆕 Usage in a New Project
+
+### Option 1: Tailwind CSS v4 (Recommended)
+
+1. Install the package and Tailwind CSS v4:
+   ```bash
+   npm install goodparty-styleguide tailwindcss@next
+   ```
```

Comment:
Should this say `Tailwind CSS >= v4`, since the command ultimately installs `@next`, not pegged to v4? 🤔